assets/plugin-template/rollup.config.js.mustache

```diff
@@ -10,5 +10,10 @@ export default {
     sourcemap: true,
     inlineDynamicImports: true,
   },
+  {
+    file: 'dist/index.js',
```

Comment:
I think this file name makes more sense to indicate the file type and that it's a "legacy" file.

```suggestion
    file: 'dist/plugin.cjs.js',
```